### PR TITLE
ticdc: add metrics doc

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

Add TiCDC metric doc and related screeshots.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5221
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/monitor-ticdc.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+The description of each metric in the **Changefeed** panel is as follows:
+
+- Changefeed table count: The number of tables that each TiCDC node needs to replicate in the replication task
+- Processor resolved ts: The timestamps that have been resolved in the TiCDC cluster
+- Table resolved ts: The replication process of each table in the replication task
+- Changefeed checkpoint: The process of replicating data to the downstream. Normally, the green bars are connected to the yellow line
+- PD etcd requests/s: The number of requests that a TiCDC node sends to PD per second
+- Exit error count: The number of errors that interrupt the replication task per minute
+- Changefeed checkpoint lag: The data replication process lag (the unit is second) between the upstream and the downstream
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:59]:
```suggestion
- Changefeed checkpoint lag: The process lag of data replication (the unit is second) between the upstream and the downstream
```

File: ticdc/monitor-ticdc.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- Changefeed resolved ts lag: The data replication process lag (the unit is second) between the upstream and TiCDC nodes
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:60]:
```suggestion
- Changefeed resolved ts lag: The process lag of data replication (the unit is second) between the upstream and TiCDC nodes
```

File: ticdc/monitor-ticdc.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+- Flush sink duration: The histogram of the time spent by TiCDC asynchronously flushing data to the downstream
+- Flush sink duration percentile: Within one second, the 95%, 99%, and 99.9% probability of the time spent by TiCDC asynchronously flushing data to the downstream
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:62]:
```suggestion
- Flush sink duration percentile: The time (P95, P99, and P999) spent by TiCDC asynchronously flushing data to the downstream within one second
```

File: ticdc/monitor-ticdc.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+- Sink write duration: The histogram of the time spent by TiCDC writing a transaction change to the downstream
+- Sink write duration percentile: Within one second, the 95%, 99%, and 99.9% probability of the time spent by TiCDC writing a transaction change to the downstream
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:64]:
```suggestion
- Sink write duration percentile: The time (P95, P99, and P999) spent by TiCDC writing a transaction change to the downstream within one second
```

File: ticdc/monitor-ticdc.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+- MySQL sink conflict detect duration: The histogram of the time spent by detecting MySQL sink conflicts
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:65]:
```suggestion
- MySQL sink conflict detect duration: The histogram of the time spent on detecting MySQL sink conflicts
```

File: ticdc/monitor-ticdc.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+- MySQL sink conflict detect duration percentile: Within one second, the 95%, 99%, and 99.9% probability of the time spent by detecting MySQL sink conflicts
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:66]:
```suggestion
- MySQL sink conflict detect duration percentile: The time (P95, P99, and P999) spent on detecting MySQL sink conflicts within one second
```

File: ticdc/monitor-ticdc.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+- MySQL sink worker load: The workload of MySQL sink workers of TiCDC nodes
+
+## Events
+
+Following is an example of the **Events** panel:
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:71]:
```suggestion
The following is an example of the **Events** panel:
```

File: ticdc/monitor-ticdc.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+
+![TiCDC Dashboard - Events metrics 2](/media/ticdc/ticdc-dashboard-events-1.png)
+![TiCDC Dashboard - Events metrics 2](/media/ticdc/ticdc-dashboard-events-2.png)
+![TiCDC Dashboard - Events metrics 2](/media/ticdc/ticdc-dashboard-events-3.png)
+
+The description of each metric in the **Events** panel is as follows:
+
+- Eventfeed count: TiCDC 节点中 Eventfeed RPC 的个数 The number of Eventfeed RPC of TiCDC nodes
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:79]:
```suggestion
- Eventfeed count: The number of Eventfeed RPC requests of TiCDC nodes
```

File: ticdc/monitor-ticdc.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+- Event size percentile: Within one second, the 95%, 99%, and 99.9% probability of the event size which TiCDC receives from TiKV
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:80]:
```suggestion
- Event size percentile: The event size (P95, P99, and P999) which TiCDC receives from TiKV within one second
```

File: ticdc/monitor-ticdc.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+- Eventfeed error/m: The number of errors reported by Eventfeed RPC of TiCDC nodes per minute
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:81]:
```suggestion
- Eventfeed error/m: The number of errors reported by Eventfeed RPC requests of TiCDC nodes per minute
```

File: ticdc/monitor-ticdc.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+- KV client receive events/s: The number of events that the KV client module of TiCDC nodes receives from TiKV per second
+- Puller receive events/s: The number of events that the Puller module of TiCDC nodes receives from the KV client per second
+- Puller output events/s: The number of events that the Puller module of TiCDC nodes sends to the Sorter module per second
+- Sink flush rows/s: The number of events that TiCDC nodes write to the downstream per second
+- Puller buffer size: The number of events that TiCDC nodes cache in the Puller module
+- Entry sorter buffer size: The number of events that TiCDC nodes cache in the Sorter module
+- Processor/Mounter buffer size: The number of events that TiCDC nodes cache in the Processor module and the Mounter module
+- Sink row buffer size: The number of events that TiCDC nodes cache in the Sink module
+- Entry sorter sort duration: TiCDC The histogram of the time spent by TiCDC nodes sorting events
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:90]:
```suggestion
- Entry sorter sort duration: The histogram of the time spent by TiCDC nodes sorting events
```

File: ticdc/monitor-ticdc.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+- Entry sorter sort duration percentile: Within one second, the 95%, 99%, and 99.9% probability of the time spent by TiCDC sorting events
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:91]:
```suggestion
- Entry sorter sort duration percentile: The time (P95, P99, and P999) spent by TiCDC sorting events within one second
```

File: ticdc/monitor-ticdc.md [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+- Entry sorter merge duration: The histogram of the time spent by TiCDC nodes merging sorted events
+- Entry sorter merge duration percentile: Within one second, the 95%, 99%, and 99.9% probability of the time spent by TiCDC merging sorted events
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:93]:
```suggestion
- Entry sorter merge duration percentile: The time (P95, P99, and P999) spent by TiCDC merging sorted events within one second
```

File: ticdc/monitor-ticdc.md [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+- Mounter unmarshal duration: The histogram of the time spent by TiCDC nodes unmarshaling events
+- Mounter unmarshal duration percentile: Within one second, the 95%, 99%, and 99.9% probability of the time spent by TiCDC unmarshaling events
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:95]:
```suggestion
- Mounter unmarshal duration percentile: The time (P95, P99, and P999) spent by TiCDC unmarshaling events within one second
```

File: ticdc/monitor-ticdc.md [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+- KV client dispatch events/s: The number of events that the KV client module dispatches among the TiCDC nodes
+- KV client batch resolved size: The batch size of resolved timestamp messages that TiKV sends to TiCDC
+
+## TiKV
+
+Following is an example of the **TiKV** panel:
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:101]:
```suggestion
The following is an example of the **TiKV** panel:
```

File: ticdc/monitor-ticdc.md [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+
+![TiCDC Dashboard - TiKV metrics 1](/media/ticdc/ticdc-dashboard-tikv-1.png)
+![TiCDC Dashboard - TiKV metrics 2](/media/ticdc/ticdc-dashboard-tikv-2.png)
+
+The description of each metric in the **TiKV** panel is as follows:
+
+- CDC endpoint CPU: The CPU usage of the CDC endpoint threads of TiKV nodes
+- CDC worker CPU: The CPU usage of the CDC worker threads of TiKV nodes
+- Min resolved ts: The minimum resolved timestamp on TiKV nodes
+- Min resolved region: The minimum region ID of resolved timestamp on TiKV nodes
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:111]:
```suggestion
- Min resolved region: The minimum Region ID of resolved timestamps on TiKV nodes
```

File: ticdc/monitor-ticdc.md [hunk 17]
```diff
@@ -0,0 +1,117 @@
...
+- Resolved ts lag duration percentile: The lag between the minimum resolved timestamp on TiKV nodes and the current time
+- Initial scan duration: The histogram of the time spent by incremental scan when TiKV nodes connect to TiCDC nodes
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:113]:
```suggestion
- Initial scan duration: The histogram of the time spent on incremental scan when TiKV nodes connect to TiCDC nodes
```

File: ticdc/monitor-ticdc.md [hunk 18]
```diff
@@ -0,0 +1,117 @@
...
+- Initial scan duration percentile: Within one second, the 95%, 99%, and 99.9% probability of the time spent by the incremental scan of TiKV nodes
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:114]:
```suggestion
- Initial scan duration percentile: The time (P95, P99, and P999) spent on the incremental scan of TiKV nodes within one second
```

File: ticdc/monitor-ticdc.md [hunk 19]
```diff
@@ -0,0 +1,117 @@
...
+- Memory without block cache: The memory usage of TiKV nodes, not including the RocksDB block cache
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:115]:
```suggestion
- Memory without block cache: The memory usage of TiKV nodes excluding the RocksDB block cache
```

File: ticdc/monitor-ticdc.md [hunk 20]
```diff
@@ -0,0 +1,117 @@
...
+
+![TiCDC Dashboard - Changefeed metrics 1](/media/ticdc/ticdc-dashboard-changefeed-1.png)
+![TiCDC Dashboard - Changefeed metrics 2](/media/ticdc/ticdc-dashboard-changefeed-2.png)
+![TiCDC Dashboard - Changefeed metrics 3](/media/ticdc/ticdc-dashboard-changefeed-3.png)
+
+The description of each metric in the **Changefeed** panel is as follows:
+
+- Changefeed table count: The number of tables that each TiCDC node needs to replicate in the replication task
+- Processor resolved ts: The timestamps that have been resolved in the TiCDC cluster
+- Table resolved ts: The replication process of each table in the replication task
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:55]:
```suggestion
- Table resolved ts: The replication progress of each table in the replication task
```

File: ticdc/monitor-ticdc.md [hunk 21]
```diff
@@ -0,0 +1,117 @@
...
+- Changefeed checkpoint: The progress of replicating data to the downstream. Normally, the green bars are connected to the yellow line
+- PD etcd requests/s: The number of requests that a TiCDC node sends to PD per second
+- Exit error count: The number of errors that interrupt the replication task per minute
+- Changefeed checkpoint lag: The process lag of data replication (the unit is second) between the upstream and the downstream
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:59]:
```suggestion
- Changefeed checkpoint lag: The progress lag of data replication (the unit is second) between the upstream and the downstream
```

File: ticdc/monitor-ticdc.md [hunk 22]
```diff
@@ -0,0 +1,117 @@
...
+- Changefeed resolved ts lag: The process lag of data replication (the unit is second) between the upstream and TiCDC nodes
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:60]:
```suggestion
- Changefeed resolved ts lag: The progress lag of data replication (the unit is second) between the upstream and TiCDC nodes
```

File: ticdc/monitor-ticdc.md [hunk 23]
```diff
@@ -0,0 +1,117 @@
...
+
+![TiCDC Dashboard - TiKV metrics 1](/media/ticdc/ticdc-dashboard-tikv-1.png)
+![TiCDC Dashboard - TiKV metrics 2](/media/ticdc/ticdc-dashboard-tikv-2.png)
+
+The description of each metric in the **TiKV** panel is as follows:
+
+- CDC endpoint CPU: The CPU usage of the CDC endpoint threads of TiKV nodes
+- CDC worker CPU: The CPU usage of the CDC worker threads of TiKV nodes
+- Min resolved ts: The minimum resolved timestamp on TiKV nodes
+- Min resolved region: The minimum Region ID of resolved timestamps on TiKV nodes
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:111]:
```suggestion
- Min resolved region: The Region ID of the minimum resolved timestamps on TiKV nodes
```

File: ticdc/monitor-ticdc.md [hunk 24]
```diff
@@ -0,0 +1,117 @@
...
+
+The following is an example of the **TiKV** panel:
+
+![TiCDC Dashboard - TiKV metrics 1](/media/ticdc/ticdc-dashboard-tikv-1.png)
+![TiCDC Dashboard - TiKV metrics 2](/media/ticdc/ticdc-dashboard-tikv-2.png)
+
+The description of each metric in the **TiKV** panel is as follows:
+
+- CDC endpoint CPU: The CPU usage of the CDC endpoint threads of TiKV nodes
+- CDC worker CPU: The CPU usage of the CDC worker threads of TiKV nodes
```
Comment (TomShawn) [inline ticdc/monitor-ticdc.md:109]:
```suggestion
- CDC endpoint CPU: The CPU usage of the CDC endpoint threads on TiKV nodes
- CDC worker CPU: The CPU usage of the CDC worker threads on TiKV nodes
```

**Review by TomShawn — COMMENTED** [review]
Rest LGTM
@overvenus PTAL

**Comment by overvenus** [comment]
LGTM, thanks!

**Review by TomShawn — APPROVED** [review]

**Pull request merged**